src/data/queries/index.ts

```diff
@@ -1,51 +1,51 @@
 import { createQueries } from 'next-drupal-query'
-import * as PressRelease from './pressRelease'
```

Comment:
For my own clarity: I just copied the before and after into files, sorted the before, and did a diff against the after. They're all the same imports, just alphabetized.